config/pagerduty.spc

```diff
@@ -0,0 +1,11 @@
+connection "pagerduty" {
+  plugin = "pagerduty"
+  
+  # Account/user API token authentication
+  # This can also be set via the `PAGERDUTY_TOKEN` environment variable.
+  # token = "u+AtBdqvNtestTokeNcg"
+
+  # A custom proxy endpoint as PagerDuty client API URL overriding 'service_region' setup.
+  # Default is "https://api.pagerduty.com"
+  # api_url_override = "YOUR_CUSTOM_ENDPOINT"
```

Comment:
@cbruno10 we don't need this arg if the endpoint is same as default
If `api_url_override` is not defined, it will fallback to default endpoint, i.e. `https://api.pagerduty.com`

Reply:
@Subhajit97 How likely is it that users would need to use a different endpoint? We can support this, but if it's very unlikely for users to change it, we can add it when there's a specific use case.